docs/tutorials/using_apis/Mbed_USB_WAV_Audio_Player/Mbed_USB_WAV_Audio_Player.md

```diff
@@ -16,39 +20,52 @@ This tutorial uses the USBAudio class of the USB library to send audio data from
 
```

Comment:
Not a big fan of this heading "What you need".
Wouldn't it be better to call it "Requirements"

Reply:
👍